src/components/dropdowns/_dropdowns.scss

```diff
@@ -64,26 +79,29 @@ $include-html: false !default;
       align-items: center;
       justify-content: flex-start;
       height: 100%;
-      padding: $layoutDefaultPadding/4 $dropdownHeight/2;
+      padding: $layoutDefaultPadding / 4 $dropdownHeight / 2;
       white-space: nowrap;
       overflow: hidden;
       text-decoration: none;
 
       &:hover,
       &:active {
-        background-color: $grayAltLight;
+        background-color: $dropdownBorderColor;
       }
     }
 
     &--opened {
-      background-color: $white;
-      border-color: $grayPrimary;
-      height: auto;
+      background-color: $dropdownBackground;
+      border-color: $dropdownOpenedBorderColor;
+      overflow: visible;
+      border-radius: 16px 16px 0 0;
+      border-bottom-color: transparent;
 
       &:hover,
       &:active {
-        background-color: $white;
-        border-color: $grayPrimary;
+        background-color: $dropdownBackground;
+        border-color: $dropdownOpenedBorderColor;
+        border-bottom-color: transparent;
```

Comment:
it overides styles for :hover,:active when there is no '--opened';
i has to be like that my dear